### PR TITLE
fix(curriculum): update object example in Using Objects for Lookups

### LINKED ISSUE #48387: Possibe error in code example
### Describe the Issue

The example shows "const thirdLetter = alpha[2]". Below that, the snippet of text says that alpha[2] should be "Y".
"thirdLetter is the string Y, lastLetter is the string C, and valueLookup is the string Y."

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups

```js

const alpha = {
  1:"Z",
  2:"Y",				//<--------------- thirdLetter ??
  3:"X",
  4:"W",
  ...
  24:"C",				//<--------------- lastLetter ??
  25:"B",
  26:"A"
};

const thirdLetter = alpha[2];
const lastLetter = alpha[24];

const value = 2;
const valueLookup = alpha[value];

```

The snippet has the below explanation in the instructions:

```markdown

`thirdLetter` is the string `Y`, lastLetter is the string `C`, and `valueLookup` is the string `Y`.
```

### Expected behavior

I believe correcting the const "thirdLetter" to "secondLetter" could be less confusing.

### PR DESCRIPTION
**Pull request by nayabatir1**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48387

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md [hunk 1]
```diff
@@ -14,25 +14,23 @@ Objects can be thought of as a key/value storage, like a dictionary. If you have
...
-  4:"W",
-  ...
-  24:"C",
-  25:"B",
-  26:"A"
+const article = {
+  "title": "How to create objects in JavaScript",
+  "link": "https://www.freecodecamp.org/news/a-complete-guide-to-creating-objects-in-javascript-b0e2450655e8/",
+  "author": "Kaashan Hussain",
+  "language": "JavaScripts",
```
Comment (raisedadead) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:21]:
```suggestion
  "language": "JavaScript",
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by raisedadead — APPROVED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md [hunk 2]
```diff
@@ -14,25 +14,23 @@ Objects can be thought of as a key/value storage, like a dictionary. If you have
 Here is an example of a simple reverse alphabet lookup:
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:14]: Need a verbage update here too

Comment (nayabatir1) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:14]: will this work `Here is an example of a simple reverse article lookup:` ?

Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:14]: No "reverse".
 
> Here is an example of an article object:

**Review by jeremylt — APPROVED** [review]

**Pull request merged**